local_rl_loop.py

```diff
@@ -34,15 +34,17 @@ def rl_loop():
         'k': 8, 'fc_width': 16, 'num_shared_layers': 1, 'l2_strength': 1e-4, 'momentum': 0.9}
 
     dual_net.TRAIN_BATCH_SIZE = 16
+    dual_net.SHUFFLE_BUFFER_SIZE = 10000
 
     #monkeypatch the shuffle buffer size so we don't spin forever shuffling up positions.
```

Comment:
comment appears out of order now?

Reply:
Done.

Reply:
Are you thinking 1 plot per generation? Or 1 plot that measures all prior generations in predicting this generations self play data?